src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -159,7 +164,11 @@ public SeekableByteChannelPrefetcher(SeekableByteChannel chan, int bufSize) thro
             this.bufSize = bufSize;
         }
         this.open = true;
-        exec = Executors.newFixedThreadPool(1);
+        // make sure the prefetching thread's name indicate what it is and
+        // which prefetcher it belongs to (for debugging purposes only, naturally).
+        String nameFormat = "nio-prefetcher-" + prefetcherIndex + "-thread-%d";
```

Comment:
I'm not sure what you're asking or trying to accomplish. You mean to store it beyond the scope of the constructor?